### ISSUE TITLE
The CSV Commissions Export on [edd_commissions] is broken

### ISSUE THREAD

**Issue by chriscct7** [easydigitaldownloads/EDD-Commissions#34]
Haven't nailed the cause down yet

**Comment by pippinsplugins** [comment]
In what way does it break? Is it coming up blank?

**Comment by cklosowski** [comment]
Yes, please give us a little more to go on...a better description than 'Broken' might lend us to have an idea of what exactly is wrong.

**Comment by chriscct7** [comment]
They don't work. Doesn't do anything. Redirects to the home url. I was thinking I would solve it last night. My guess is that the edd_action isn't firing for some reason.

**Comment by pippinsplugins** [comment]
Got another report of this today. Tracking it down now.

**Comment by pippinsplugins** [comment]
It's working for me locally.

**Comment by pippinsplugins** [comment]
It also works fine for me on the EDD live site. Closing as invalid.

**Comment by chriscct7** [comment]
This doesn't work in many places. For sure doesn't work when the shortcode is used on the admin and I've seen several sites where this doesn't work on the frontend either. This doesn't work: http://screencloud.net/v/gKb5. There is an underlying problem here, and it almost certainly is that edd_action isn't firing

**Comment by chriscct7** [comment]
If you need a simple replication, install FES and generate some commissions for a user and then try to download the CSV using the earnings tab on the vendor dashboard. Both csvs on the frontend and backend are broken, and all they do is echo the shortcode

**Comment by pippinsplugins** [comment]
That's an FES bug if it doesn't work there. I tested default dashboard and
frontend page. Both work perfectly.

try it yourself out side of FES. If it fails then, re-open.

**Comment by chriscct7** [comment]
FES simply does an echo of the shortcode. Therefore, if it doesn't work its because its broken in core.

Here's default 2015 theme, no plugins but EDD + comissions active, with just the shortcode on a page. Doesn't work: https://db.tt/1f3Xo86U

**Comment by chriscct7** [comment]
It's definitely the ajax handler not firing. There is a bug in EDD's edd_action thing. Its not firing when it should be to pick up the commission export

**Comment by chriscct7** [comment]
When this bug was first reported by me, it wasn't even from an FES site. We've gotten several reports on the forums that it doesn't work, many of them on non-FES sites. Most of them seem to come from FES, but as you can see above its not just FES. It just simply doesn't work for 99% of people. Frankly, EDD.com is one of the few places I know of where it works right now, so maybe that's a clue. I think it has to do with the edd_action ajax thing not firing when it's supposed to be.

**Comment by pippinsplugins** [comment]
I just tested this with FES on the Earnings tab of the vendor dashboard and it worked perfectly.

**Comment by pippinsplugins** [comment]
[This user](https://easydigitaldownloads.com/support/topic/unable-to-download-paid-commision-csv-file/) is reporting the problem so I'm going to try and log into their site to look at it closer.
